oaebu_workflows/database/schema/onix_invalid_isbn_2022-03-27.json

```
@@ -0,0 +1,8 @@
+[
+    {
+        "mode": "NULLABLE",
+        "name": "Primary_ISBN",
```

```suggestion
        "name": "ISBN13",
```

The ISBN13 field is always used for the onix data source